src/pages/Sources.js

```diff
@@ -135,15 +136,36 @@ const SourcesPage = () => {
                 actionsConfig={isOrgAdmin ? {
                     actions: [
                         <Link to={routes.sourcesNew.path} key="addSourceButton">
-                            <Button variant='primary'>
+                            <Button variant='primary' id="addSourceButton">
                                 <FormattedMessage
                                     id="sources.addSource"
                                     defaultMessage="Add source"
                                 />
                             </Button>
                         </Link>
                     ]
-                } : undefined}
+                } : {
+                    actions: [
+                        <Tooltip
+                            content={
+                                intl.formatMessage({
+                                    id: 'sources.notAdminButton',
+                                    defaultMessage: 'You do not have permission to perform this action.'
+                                })
+                            }
+                            key="addSourceButton"
+                        >
+                            <span>
```

Comment:
you should add a tab index to the span element. Otherwise, users won't be able to focus it with the keyboard and see the tooltip.

Reply:
fixed :+1: